morpheus/utils/http_utils.py

```diff
@@ -141,7 +141,7 @@ def request_with_retry(
                          request_kwargs['method'],
                          request_kwargs['url'],
                          e)
-            logger.debug("Sleeping for %s seconds before retrying request again", sleep_time)
+            logger.debug("Sleeping for %s seconds before retrying request again", sleep_time_)
```

Comment:
Lets rename this something different so its easier to distinguish. Possibly `actual_sleep_time`